src/Application/Regulation/Command/SaveRegulationOrderStorageCommandHandler.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Application\Regulation\Command;
+
+use App\Application\StorageInterface;
+use App\Domain\Regulation\Repository\StorageRegulationOrderRepositoryInterface;
+use App\Domain\Regulation\StorageRegulationOrder;
+use App\Infrastructure\Adapter\IdFactory;
+
+final class SaveRegulationOrderStorageCommandHandler
+{
+    public function __construct(
+        private StorageInterface $storage,
+        private StorageRegulationOrderRepositoryInterface $storageRegulationOrderRepository,
+        private IdFactory $idFactory,
+    ) {
+    }
+
+    public function __invoke(SaveRegulationOrderStorageCommand $command): void
+    {
+        if ($command->file !== null || $command->url !== null) {
```

Comment:
```suggestion
        if ($command->file !== null) {
```

Le cas pour la gestion de l'url à placer ailleurs comme la logique n'est pas la même que pour le `file`